index.html

```diff
@@ -2,77 +2,114 @@
 <html>
     <head>
         <meta charset="UTF-8">
-
         <title>Uniter PHP NavMenuComponent demo</title>
-
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css">
+        <script src="/assets/jquery-full.js"></script>
+        <link rel="stylesheet" href="css/bootstrap.min.css">
         <link rel="stylesheet" href="css/style.css">
     </head>
     <body>
-        <h1>Uniter PHP NavMenuComponent demo</h1>
-
-        <p>
-            The logic for this page is written entirely in PHP,
-            see <a href="//github.com/uniter/uniter-jquery/blob/master/php/src/Demo/Component/NavMenuComponent.php">NavMenuComponent</a>
-            for details.
-        </p>
 
-        <p>
-            The tests are written using PHPUnit and run on the command line (for now,)
-            see <a href="//github.com/uniter/uniter-jquery/blob/master/tests/phpunit/Demo/Tests/Component/NavMenuComponentTest.php">NavMenuComponentTest</a>.
-        </p>
+        <div class="full_row">
+            <hr />
+        </div>
 
-        <nav>
-            <ol class="primary-nav">
-                <li data-submenu="#submenu1">
-                    <a href="#">Menu 1</a>
-                    <ol class="submenu" id="submenu1">
-                        <li data-submenu="#submenu1_1">
-                            <a href="#">Menu 1 sub 1</a>
+        <div class="all_demos">
+            <h1>
+                Uniter PHP<br/>in Browser Demos
+            </h1>
+            <p>
+                The in-browser logic for all these pages is written entirely in PHP,
+                see <a target="_blank" href="//github.com/PharaohTools/uniter-jquery/blob/master/php/src/">Demos</a>
+                for a look at the php code.<br /> All Javascript and JQuery object and method calls come directly from php code.
```

Comment:
Being picky now, but `Javascript` should be `JavaScript` (and `JQuery` -> `jQuery`) 😁